modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/WatchProcessor.java

```diff
@@ -221,15 +221,15 @@ public CompletableFuture<Void> notifyWatches(long newRevision, List<Entry> updat
      * @return Updated value of {@link #notificationFuture}.
      */
     @VisibleForTesting
-    CompletableFuture<Void> enqueue(Supplier<CompletableFuture<Void>> asyncAction) {
+    CompletableFuture<Void> enqueue(Supplier<CompletableFuture<Void>> asyncAction, String additionalInfo) {
```

Comment:
I see. Can we please convert the second parameter into a supplier, so that we don't calculate these long strings every time?